apps/dashboard/src/components/contract-functions/contract-function.tsx

```diff
@@ -111,21 +118,22 @@ function ContractFunctionInner({ contract, fn }: ContractFunctionProps) {
 
   return (
     <Flex direction="column" gap={1.5}>
-      <Flex
-        alignItems={{ base: "start", md: "center" }}
-        gap={2}
-        direction={{ base: "column", md: "row" }}
-      >
+      <Flex alignItems="center" gap={2} direction="row" flexWrap="wrap">
         <Flex alignItems="baseline" gap={1} flexWrap="wrap">
           <Heading size="subtitle.md">{camelToTitle(fn.name)}</Heading>
           <Heading size="subtitle.sm" className="text-muted-foreground">
             ({fn.name})
           </Heading>
         </Flex>
-        {isFunction && (
-          <Badge size="label.sm" variant="subtle" colorScheme="green">
-            {fn.stateMutability}
-          </Badge>
+        {isFunction && <Badge variant="success">{fn.stateMutability}</Badge>}
+        {functionSelector && (
```

Comment:
Not sure how helpful the function selector is, Deferring this to @jnsdls  to decide whether this should be added or not